docs/consumers.md

```diff
@@ -504,12 +504,15 @@ A typical sequence of events would be the following:
 
  * A queue is declared and some consumers register to it at roughly the
  same time.
- * The very first registered consumer become the *single active consumer*:
+ * The very first registered consumer becomes the *single active consumer*:
  messages are dispatched to it and the other consumers are ignored.
- * The single active consumer is cancelled for some reason or simply dies.
- One of the registered consumer becomes the new single active consumer and
- messages are now dispatched to it. In other terms, the queue fails over
- automatically to another consumer.
+ * If the queue is a quorum queue, and a new consumer registers with a higher priority,
+ the queue will stop delivering messages to the current active consumer and once
+ all messages are acknowledged, the new consumer will become the active consumer.
+ * When the single active consumer is cancelled for some reason or simply dies,
```

Comment:
@mkuratczyk, I would rewrite the 1st sentence as: "If the queue is a quorum queue, and a new consumer registers with a higher priority, then the queue stops delivering messages to the current active consumer. When 
 all messages are acknowledged, the new consumer becomes the active consumer. "
 
 It should breaks it up into multiple sentences, reduces down word count and helps readability